server/src/api_endpoints/personas/permissions.js

```diff
@@ -16,11 +16,15 @@ const getPersonaPermissions = async (req, res, next) => {
 const createPersonaPermissions = async (req, res, next) => {
   try {
     const { persona_key: personaKey } = req.params
-    await checkPersonaExists(personaKey)
-    await dbAdminPersonaPermissions.insertPersonaPermissions(personaKey, req.body)
+    const permissionKeys = Array.from((new Set(req.body)))
```

Comment:
Do you like the function name or this style syntax?

```javascript
const permissionKeys = [...new Set(req.body)]
```

Reply:
The syntax looks nicer. I will update it.